### PR TITLE
Fix possibility of obfuscator secrets being supplied incorrectly

### LINKED ISSUE #7268: Make lib.obfuscator raise missingsecret exception when the secret is an empty/whitespace-only string
turns out github actions will happily use secrets not declared in the project but set them to an empty string which passes the "is not none" check in create_release

### PR DESCRIPTION
**Pull request by Miepee**
Fixes #7268

### CODE REVIEW

File: tools/create_release.py [hunk 1]
```diff
@@ -183,7 +183,7 @@ async def main():
     icon_path = randovania.get_icon_path()
     shutil.copyfile(icon_path, icon_path.with_name("executable_icon.ico"))
 
-    if (secret := os.environ.get("OBFUSCATOR_SECRET")) is not None:
+    if (secret := os.environ.get("OBFUSCATOR_SECRET")) is not None and len(secret) >= 1 and not secret.isspace():
```
Comment (henriquegemignani) [inline tools/create_release.py:186]: maybe this?
```python
secret = os.environ.get("OBFUSCATOR_SECRET", "")
if secret.trim() and not secret.isspace():
```

**Review by henriquegemignani — CHANGES_REQUESTED** [review]

File: tools/create_release.py (hunk 1 above)
Comment (Miepee) [reply tools/create_release.py:186]: What is `trim` supposed to be for a function? Do you mean strip?

Comment (henriquegemignani) [reply tools/create_release.py:186]: probably, too used to different names in different languages

Comment (Miepee) [reply tools/create_release.py:186]: doesn't work then.
Stripping an empty string, results in an empty string. and isspace only works on strings with length bigger than 0.
```py
>>> s = ""
>>> s.strip().isspace()
False
>>> 
```

Comment (henriquegemignani) [reply tools/create_release.py:186]: I did suggest both conditions?

Comment (Miepee) [reply tools/create_release.py:186]: oh, i got switched up by it. forget sometimes that a string itself can be truthy.

**Review by henriquegemignani — APPROVED** [review]

**Pull request merged**